WcaOnRails/config/i18n-tasks.yml.erb

```diff
@@ -154,6 +154,10 @@ ignore:
 <%
   require './lib/custom_wca_i18n_scanner.rb'
   I18n::Tasks.add_scanner 'CustomWcaI18nScanner', only: %w(*.erb)
+
+  I18n::Tasks.add_scanner 'I18n::Tasks::Scanners::PatternMapper',
+    only: %w(*.js),
+    patterns: [['I18nHTMLTranslate\si18nKey=%{key}', '%{key}']]
```

Comment:
Please change the pattern to use something like `\s+` (where `+` is RegExp for "one or more") such that line breaks and tab characters aren't ignored by this pattern match.